build.sbt

```diff
@@ -8,13 +8,12 @@ import scala.language.postfixOps
 
 lazy val currentVersion = "2.1.0"
```

Comment:
The next release with this change would be 2.2.0 (as it contains jackson minor upgrades)

Reply:
done